src/realtime/ash/ash_hazard.py

```diff
@@ -130,82 +159,49 @@ def copy_style(self):
     def write_keywords(self):
         keyword_io = KeywordIO()
 
+        hazard_classes = {
+            ash_hazard_classes['key']: {
+                u'active': True,
+                u'classes': {
+                    u'high': [5.0, 10.0],
+                    u'very high': [10.0, 9999.0],
+                    u'very low': [0.01, 0.1],
+                    u'medium': [2.0, 5.0],
+                    u'low': [0.1, 2.0]
+                }
+            }
+        }
+
         keywords = {
             'active_band': 1,
-            'hazard_category': u'single_event',
+            'hazard_category': hazard_category_single_event['key'],
```

Comment:
I thought the key like 'hazard_category' have definition too?

Reply:
Well, actually I had plan for that, and started working on it... But, my plan is too far (i.e. also put the same key in the metadata), and it needs much refactoring (which is exhausting but not seen). So, I decided to pause the work.... Then, for now, we need to use this way.